RedHat-UX/red-hat-design-system
# docs(status-page): Fixing status table padding

## What I did

1. Fixed the whitespace / markdown issue inside the [Design/code Status page](/design-code-status/) status tables


## Testing Instructions

1. Go to the [Design/code Status page](https://deploy-preview-1674--red-hat-design-system.netlify.app/design-code-status/) and check the padding in the table cells looks good.
2. [Compare to what's live](https://ux.redhat.com/design-code-status/)
